assets/svg_to_png.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# (1000 is the width in px)
+# produces a file named file.png in the current dir
+
+function usage_and_example {
+    echo -e "usage: ./to_png.sh SVG_FILE WIDTH\n  example: ./to_png.sh logo-text.png 666"
```

Comment:
You might enjoy using `$(basename $0)` rather than duplicating the name of the script inside the script (which also avoids it getting out of step).

Reply:
Did you miss an instance of the name? The example still says `./to_svg.sh`.